src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 300ms, transform 300ms;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2);
+    }
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
you can move it up closer to parent

Reply:
in checklist good example we have space 
`&__buy-link {
  display: flex;
  margin-top: 20px;

  &:hover {
    color: blue;
  }
}`
I think space need here